cmd/add/smtpConfiguration/smtpConfiguration.go

```diff
@@ -0,0 +1,199 @@
+				var promptRes string
+				
+				if  val == "Key" {
+					fmt.Println("(If you don't have a mailazy account. Please Create a mailazy account via https://app.mailazy.com/signup)")
+				}
+				prompt.SurveyAskOne(&survey.Input{
+					Message: cmdutil.SmtpOptionNames[val] + ":",
+				}, &promptRes, survey.WithValidator(survey.Required))
+
+				if strings.TrimSpace(promptRes) == "" {
+					return errors.New(cmdutil.SmtpOptionNames[val] + " is required")
+				}
+				if val == "FromEmailId" && !cmdutil.ValidateEmail.MatchString(promptRes)  {
+					return &cmdutil.FlagError{Err: errors.New("Invalid email format")}
+				}
+				if val == "SmtpPort" {
+					smtpPort, err := strconv.ParseInt(promptRes, 10, 64)
+					if err != nil  {
+						return &cmdutil.FlagError{Err: errors.New("Please enter the valid SMTP Port")}
+					}
+					field.SetInt(smtpPort)
+				} else {
+					field.SetString(strings.TrimSpace(promptRes))
+				}
+			}
+		}
+	}
+	if num != 9 {
+		smtpSchema.Provider = smtpProviders[num]
+	}
+	smtpSchema.IsSsl = isSsl
+	
+	if num == 0 {
+      if strings.Contains(conf.HubPageDomain ,"devhub.") {
+        smtpSchema.SmtpHost = "devsmtp.mailazy.com";
+        smtpSchema.SmtpPort = 588;
+      } else {
+		smtpSchema.SmtpHost = "smtp.mailazy.com"
+		smtpSchema.SmtpPort = 587;
+	  }
+	  smtpSchema.UserName = smtpSchema.Key
+	  smtpSchema.Password = smtpSchema.Secret
+	  smtpSchema.IsSsl = cmdutil.SmtpProviders[num].EnableSSL
+
+	  } else if num != 9 {
+		var err error
+		smtpSchema.SmtpHost = cmdutil.SmtpProviders[num].SmtpHost
+		smtpSchema.SmtpPort,err = strconv.Atoi(cmdutil.SmtpProviders[num].SmtpPort) 
+		if err != nil {
+			return err
+		}
+	}
+
+	var _, err = api.AddSMTPConfiguration(smtpSchema) 
+	if err != nil {
+		return err
+	}
+	fmt.Println("SMTP settings are saved")
+	var verify bool 
+	
+	if err := prompt.Confirm("Send an email to verify your configuration settings are correct?", 
+		&verify); err != nil {
+			return err
+	}
+	if !verify {
+		return nil
+	}
+	var emailid string
+	var verifySchema api.VerifySmtpConfigSchema
+	
+	prompt.SurveyAskOne(&survey.Input{
+		Message:  "To Email :",
+	}, &emailid, survey.WithValidator(survey.Required))
+
+	if !cmdutil.ValidateEmail.MatchString(emailid)  {
+		return &cmdutil.FlagError{Err: errors.New("Invalid email format")}
+	}
+	
+	var respMap map[string]string
+	data, _ := json.Marshal(smtpSchema)
+	json.Unmarshal(data, &respMap)
+
+	for _, val := range smtpLabels {
+		if val != "IsSsl" && val != "SmtpPort" {
+			configObj := reflect.ValueOf(&verifySchema).Elem()
+			field := configObj.FieldByName(val)
+			field.SetString(respMap[val])
+		}
+	}
+
+	verifySchema.EmailId = emailid
+	verifySchema.Message = "This is the test email to validate your SMTP credentials for LoginRadius' User Registration feature on your website. <br><br>The SMTP server credentials are verified.<br><br>Thank you,<br>LoginRadius Team"
+	verifySchema.Subject = "Test Email - LoginRadius"
```

Comment:
Better to store it in constant file and use it from there, instead of writing is directly from here

Reply:
Notes.